rocky/rocky/views/ooi_report.py

```diff
@@ -33,12 +33,12 @@
     keiko_client,
 )
 from rocky.views.finding_list import FindingListView
-from rocky.views.mixins import OOIBreadcrumbsMixin, SingleOOITreeMixin
-from rocky.views.ooi_view import BaseOOIDetailView
+from rocky.views.mixins import SingleOOITreeMixin
+from rocky.views.ooi_view import OOIBreadcrumbsMixin
 
 
 @class_view_decorator(otp_required)
-class OOIReportView(OOIBreadcrumbsMixin, BaseOOIDetailView):
+class OOIReportView(OOIBreadcrumbsMixin):
```

Comment:
Agreed that `OOIBreadcrumbsMixin` should not be a view, but it actually already was (inheriting `OrganizationView`). It also assumed existence of `self.ooi` however, so using this base class made more sense to me in that context, also because this caused views to inherit OrganizationView through other paths as well as was made clear by the diagram. 

Given the current structure of the diagram in the description after a small update I made, I think it makes more sense to let `BaseOOIDetailView` implement the `BreadcrumbsMixin` and implement the `build_breadcrumbs` logic currently implemented in `OOIBreadcrumbsMixin`. We can then actually remove the `OOIBreadcrumbsMixin`.

(P.S. The move was to avoid circular imports.)

Reply:
Done :+1: